rsts/community/contribute.rst

```diff
@@ -415,48 +415,31 @@ that integrates all Flyte components into a single binary.
 **4. Build single binary with your own code.**
 
 
-The following instructions provide guidance on how to build single binary with your customized code, using ``flyteadmin`` as an example.
+The following instructions provide guidance on how to build single binary with your customized code under the ``flyteadmin`` as an example.
 
 
 - **Note** Although we'll use ``flyteadmin`` as an example, these steps can be applied to other Flyte components or libraries as well.
   ``{flyteadmin}`` below can be substituted with other Flyte components/libraries: ``flyteidl``, ``flyteplugins``, ``flytepropeller``, ``datacatalog``, or ``flytestdlib``.
-
-- **Note** If modifications are needed in multiple components/libraries, the steps will need to be repeated for each component/library.
+- **Note** If you want to learn how flyte compiles those components and replace the repositories, you can study how ``go mod edit`` works.
 
 .. code:: shell
 
-   # Step1: Fork and clone the {flyteadmin} repository, modify the source code accordingly.
-   git clone https://github.com/flyteorg/flyteadmin.git
-   cd flyteadmin
-
-   # Step2.1: {Flyteadmin} uses Go 1.19, so make sure to switch to Go 1.19.
+   # Step1: Install Go. Flyte uses Go 1.19, so make sure to switch to Go 1.19.
    export PATH=$PATH:$(go env GOPATH)/bin
    go install golang.org/dl/go1.19@latest
    go1.19 download
    export GOROOT=$(go1.19 env GOROOT)
    export PATH="$GOROOT/bin:$PATH"
- 
-   # Step2.2: You may need to install goimports to fix lint errors.
+
+   # You may need to install goimports to fix lint errors.
    # Refer to https://pkg.go.dev/golang.org/x/tools/cmd/goimports
    go install golang.org/x/tools/cmd/goimports@latest
    export PATH=$(go env GOPATH)/bin:$PATH
 
-   # Step 3.1: Review the go.mod file in the {flyteadmin} directory to identify the Flyte components/libraries that {flyteadmin} relies on.
-   # If you have modified any of these components/libraries, use `go mod edit -replace` in the {flyteadmin} repo to replace original components/libraries with your customized ones.
-   # For instance, if you have also modified `flytepropeller`, run the following commands:
-   go mod edit -replace github.com/flyteorg/flytepropeller=/home/ubuntu/flytepropeller #replace with your own local path to flytepropeller
+   # Step2: Go to the {flyteadmin} repository, modify the source code accordingly.
+   cd flyte/flyteadmin
 
-   # Step 3.2: Generate code, fix lint errors and run unit tests for {flyteadmin}.
-   # Note, flyteidl does not have unit tests, so you can skip the `make test_unit` command.
-   # Note, flytestdlib only have `make generate` command.
-   make generate
```

Comment:
@Future-Outlier I think we still need to execute the following commands:
```shell
make generate
make lint
make test_unit
```
For instance, to generate the protobuf code, I guess we still need to run `make generate` under flyteidl folder? Correct me if I am wrong.